.phan/stubs/wordpress-constants.php

```diff
@@ -24,11 +23,15 @@
 define( 'ABSPATH', './' );
 define( 'WP_DEBUG', (bool) $v );
 define( 'WP_DEBUG_LOG', (bool) $v );
-define( 'WP_PLUGIN_DIR', './' );
-define( 'WPMU_PLUGIN_DIR', './' );
-define( 'EMPTY_TRASH_DAYS', 30 * 86400 );
 define( 'SCRIPT_DEBUG', (bool) $v );
-define( 'WP_LANG_DIR', './' );
+define( 'WPINC', 'wp-includes' );
+define( 'WP_CONTENT_DIR', ABSPATH . 'wp-content' );
+define( 'WP_PLUGIN_DIR', WP_CONTENT_DIR . '/plugins' );
```

Comment:
Noting that these have [long been discouraged](https://codex.wordpress.org/Determining_Plugin_and_Content_Directories) for use, though perhaps that's more of a PHPCS-type check (and as noted here, there's not a good alternative function).

Reply:
Thanks, I was looking for information like that link and wasn't able to find it anywhere.